docker-compose.yml

```diff
@@ -1,22 +1,25 @@
-version: '2.1'
+version: "3"
 services:
   spike-service:
+    build: .
     image: spike-service:latest
     ports:
-      - '8080:8080'
+      - "8080:8080"
     environment:
-      AUDIENCE: "kartoffel"
-      GRANT_TYPE: "client_credentials"
-      SPIKE_CLIENT_ID: "EPBpjCiczupCXZgBEq3ARZCinUt4LSXtOsVrI1Ub"
-      SPIKE_CLIENT_SECRET: "u5qxEK0sy17ReZ7t8aVeN1gCcdI9hRiM78i9J_9Z6jA2YWcvbgczcchKqZ4WnXOsdT6jxmuVCLC3~RXuVcE5EsOG4JJi~B8AJQSB"
-      REDIS_URL: 'redis://redis:6379'
-      SPIKE_TOKEN_URL: "https://spike.westeurope.cloudapp.azure.com:1337/oauth2/token"
-      SPIKE_PUBKEY_PATH: 'https://spike.westeurope.cloudapp.azure.com:1337/.well-known/publickey.pem'
-      SPIKE_PUBLIC_KEY_FULL_PATH: "/usr/src/app/utils/publickey.pem"
-      NODE_TLS_REJECT_UNAUTHORIZED: 0
+      AUDIENCE: ${KARTOFFEL_AUDIENCE}
+      GRANT_TYPE: ${GRANT_TYPE}
+      SPIKE_CLIENT_ID: ${SPIKE_CLIENT_ID}
+      SPIKE_CLIENT_SECRET: ${SPIKE_CLIENT_SECRET}
+      REDIS_URL: ${REDIS_URL}
+      SPIKE_TOKEN_URL: ${SPIKE_TOKEN_URL}
+      SPIKE_PUBKEY_PATH: ${SPIKE_PUBKEY_PATH}
+      SPIKE_PUBLIC_KEY_FULL_PATH: ${SPIKE_PUBLIC_KEY_FULL_PATH}
+      NODE_TLS_REJECT_UNAUTHORIZED: ${NODE_TLS_REJECT_UNAUTHORIZED}
+      SS_PROT: ${PORT}
```

Comment:
🦜 
Also, 'port' seems too generic for an .env file which will contain envars for a lot of services.

Reply:
it is so generic because it used in all the services haha (8080)

Reply:
So I'd call it `SERVICE_PORT` or something of this sort